FateGrandAutomata/AndroidImpl.cs

```diff
@@ -66,6 +66,11 @@ public void Highlight(Region Region, TimeSpan Timeout)
         {
             HighlightView.AddRegion(Region);
 
+            // We can't draw over the notch area
+            var cutoutAppliedRegion = CutoutManager.GetCutoutAppliedRegion(_accessibilityService);
+            Region.X -= cutoutAppliedRegion.X;
```

Comment:
Don't you need to clone the Region first?

Reply:
It's a region generated from transform, so, it's already cloned.

I've kept `Region` as a `class` when ideally it should've been a `struct`, so this works, but, I should do this before `AddRegion` for clarity.